### PR TITLE
feat: add pdm

### PR DESCRIPTION
**Pull request by huxuan**
Per discussion here: https://github.com/renovatebot/renovate/discussions/27401

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: Dockerfile [hunk 1]
```diff
@@ -61,6 +61,9 @@ RUN install-tool pipenv 2023.12.1
 # renovate: datasource=github-releases packageName=python-poetry/poetry
 RUN install-tool poetry 1.8.2
 
+# renovate: datasource=pypi
```
[the remark this answers is not in the document]
Comment (huxuan) [reply Dockerfile:61]: Done

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
can you open another PR to backport to v2, wich is currently used by official renovate images?

**Comment by huxuan** [comment]
> can you open another PR to backport to v2, wich is currently used by official renovate images?

It's my pleasure, but could you explain a little bit about how to backport? I am not so familiar with renovate actually.

**Review by viceice — APPROVED** [review]

**Comment by viceice** [comment]
> > can you open another PR to backport to v2, wich is currently used by official renovate images?
> 
> It's my pleasure, but could you explain a little bit about how to backport? I am not so familiar with renovate actually.

Simply open a new PR with same changes for `maint/2.x` branch 🤗

**Pull request merged**